### ISSUE TITLE
Not good result

### ISSUE THREAD

**Issue by dugduy** [yuchen071/DQN-for-LunarLander-v2#2]
uhm, i also try to train this based on pytorch dqn tutorial.
But the problem is that it is not stable.Eg: at the episode of 1000, some time it did pretty good, and others were too bad!!!
i see your plot results and i think you also have that problem?

**Comment by yuchen071** [comment]
Testing different parameters certainly would help!
Did you train with my parameters? Or did you use your own?
Batch sizes, learning rate, episodes, target score for early stopping, and gamma all have an effect on it
Play with those and train until it you have fairly consistent landings
You might have just landed on a bad training output
1000 episodes might not be enough too
Good luck!
